### ISSUE TITLE
Move to standard ctapipe data format for dl1+

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/cta-lstchain#972]
For the next large production, we should try hard to switch to the standard dl1 and dl2 format used in ctapipe.

While I think we will not yet be able to just use the `ctapipe-process` tool for several reasons (e.g. simultaneous muon analysis) we should be able to switch to using `ImageProcessor` and `DataWriter` in a only slightly modified version of the process tool.

This will remove a lot of lstchain's code and bring us closer to the cta data model and format.

It will also add proper support for units and the space saving column transformations.

In short, I think we'll need the following things:

- Create a `Component` for the custom LST cleaning
- Create a `Component` for the muon analysis
- Create a `LSTProcessTool`
- Switch dl2 and higher tools from `pd.read_hdf` to `ctapipe.io.read_table`.
